src/conventional/metar_csv2ioda.py

```diff
@@ -25,12 +26,51 @@
     IODA_CONV_PATH = Path(__file__).parent/'..'/'lib-python'
 sys.path.append(str(IODA_CONV_PATH.resolve()))
 
-import meteo_utils
-import ioda_conv_ncio as iconv
+import ioda_conv_engines as iconv
+from collections import defaultdict, OrderedDict
 from orddicts import DefaultOrderedDict
+import meteo_utils
 
 os.environ["TZ"] = "UTC"
 
+locationKeyList = [
+    ("station_id", "string"),
+    ("latitude", "float"),
+    ("longitude", "float"),
+    ("station_elevation", "float"),
+    ("height", "float"),
+    ("datetime", "string"),
+    ("unixtime", "integer"),
+]
+
+obsvars = {
+    'ob_temp': 'air_temperature',
+    'ob_spfh': 'specific_humidity',
+    'ob_psfc': 'surface_pressure',
+    'ob_uwnd': 'eastward_wind',
+    'ob_vwnd': 'northward_wind',
+}
+
+obsvars_units = ['K', 'kg kg-1', 'Pa', 'm s-1', 'm s-1']
```

Comment:
This is a little risky since you have to make sure that the order lines up with what you get when you loop through the obsvars dictionary. Instead you might want to do something like this:
```
obsvars = {
    'ob_temp' : [ 'air_temperature', 'K' ],
    'ob_spfh' : [ 'specific_humidity, 'kg kg-1' ],
...
}
```

Then at line 97:
```
for [ iodavar, iodaunits ] in obsvars.values():
    self.varDict[iodavar]['valKey'] = iodavar, iconv.OvalName()
    ...
    self.varAttrs[iodavar, iconv.OvalName()]['units'] = iodaunits
    ...
```

which keeps the variable names and units tied together.